examples/with-clerk/pages/index.js

```diff
@@ -40,16 +40,15 @@ const SignupLink = () => (
   </Link>
 )
 
-const apiSample = `import { withSession } from '@clerk/nextjs/api'
+const apiSample = `import { getAuth } from "@clerk/nextjs/server";
```

Comment:
```suggestion
const apiSample = `import { getAuth } from '@clerk/nextjs/server'
```
To preserve the formatting.